packages/react-data-grid/src/HeaderRow.js

```diff
@@ -3,8 +3,8 @@ const shallowEqual    = require('shallowequal');
 const BaseHeaderCell        = require('./HeaderCell');
 const getScrollbarSize  = require('./getScrollbarSize');
 const columnUtils  = require('./ColumnUtils');
-const SortableHeaderCell    = require('./cells/headerCells/SortableHeaderCell');
-const FilterableHeaderCell  = require('./cells/headerCells/FilterableHeaderCell');
+const SortableHeaderCell    = require('common/cells/headerCells/SortableHeaderCell');
```

Comment:
Any reason we moved `SortableHeaderCell` and `FilterableHeaderCell` components to the `common` folder

Reply:
They are currently referenced in both addons and main package. I didnt want to make more changes in the code so moved them to common where they are still referenced by both packages